crates/red_knot_python_semantic/resources/mdtest/unary/instance.md

```diff
@@ -0,0 +1,23 @@
+# Unary Operations
+
+```py
+class Number:
```

Comment:
This test is great! Can we also add a test for a class that implements none of these dunders, and assert that we get an "unsupported operation" diagnostic from each operator? (Unless you decide not to implement that in this PR.)